ios/FitCat.xcodeproj/xcshareddata/xcschemes/FitCat.xcscheme

```diff
@@ -66,7 +66,7 @@
       </AdditionalOptions>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "Release"
```

Comment:
Pretty sure that was why yeah. After we merge this PR we need to test the builds again because there was some weirdness like this.